modules/vaos/app/services/vaos/appointment_requests_service.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require_relative '../vaos/concerns/headers'
+
+module VAOS
+  class AppointmentRequestsService < Common::Client::Base
+    include Common::Client::Monitoring
+    include VAOS::Headers
+
+    configuration VAOS::Configuration
+
+    attr_accessor :user
+
+    STATSD_KEY_PREFIX = 'api.vaos'
+
+    def self.for_user(user)
+      rs = VAOS::AppointmentRequestsService.new
+      rs.user = user
+      rs
+    end
+
+    def get_requests(start_date = nil, end_date = nil)
+      with_monitoring do
+        response = perform(:get, get_appointment_requests_url(start_date, end_date), headers(user))
+
+        {
+          data: response.body[:appointment_requests].map { |request| OpenStruct.new(request) },
+          meta: pagination
+        }
+      end
+    rescue Common::Client::Errors::ClientError => e
+      raise_backend_exception('VAOS_502', self.class, e)
+    end
+
+    private
+
+    def get_appointment_requests_url(start_date = nil, end_date = nil)
+      url = '/var/VeteranAppointmentRequestService/v4/rest/appointment-service'
+      url += "/patient/ICN/#{user.icn}/appointments"
+      url += get_date_url(start_date, end_date).empty? ? '' : "?#{get_date_url(start_date, end_date)}"
```

Comment:
i think i need to call .to_query on this... forgot. specs only passing because VCR, but this would likely blow up the actual request -- might need to manually test this on staging to validate what params are required which are optional.